src/ParseUser.js

```diff
@@ -506,6 +506,28 @@ class ParseUser extends ParseObject {
     });
   }
 
+  /**
+   * Verify whether a given password is the password of the current user.
+   *
+   * @param {String} password A password to be verified
+   * @param {Object} options
+   * @return {Promise} A promise that is fulfilled with a user
+   *  when the password is correct.
+   */
+  verifyPassword(password: string, options?: RequestOptions): Promise<ParseUser> {
```

Comment:
Can you explain why is there a need for `options` on most methods? I had to it since I saw others had it too although I first tried to implement without it and works just ffine.

Reply:
Most methods have options to support ACL's, installationId etc and allow for usability in the future